### PR TITLE
add reference to xpath functions

### PR DESCRIPTION
**Pull request by newtewt**
I think this probably could be better but is very bare minimum. I couldn't think of better words this morning to describe what the functions do.  A more in depth ticket about using functions, operators, and maybe a guide on using calculations might be a future ticket.

### CODE REVIEW

**Review by njogz — APPROVED** [review]
Looks good.

File: content/en/apps/reference/forms/app.md [hunk 1]
```diff
@@ -59,7 +59,13 @@ Since writing raw XML can be tedious, we suggest creating the forms using the [X
...
+We use [medic/openrosa-xpath-evaluator](https://github.com/medic/openrosa-xpath-evaluator) to evaluate xpath in Enketo forms. One non-standard feature is that you can concatenate string values with either the `+` operator but app builders are strongly discouraged from doing this as this feature has been deprecated and will be removed in a future version. 
+
+To support more complex actions we also support [ODK Xpath Functions](https://getodk.github.io/xforms-spec/#xpath-functions) and [ODK XPath operators](https://getodk.github.io/xforms-spec/#xpath-operators). These allow you to perform programatic tasks in your forms. 
+
+Read more about the [XPath specification](https://www.w3.org/TR/xpath/)
```
Comment (abbyad) [inline content/en/apps/reference/forms/app.md:66]: While we are here taking the opportunity to clarify and revise older text:

```suggestion
Calculations are achieved within app forms using XPath statements in the "calculate" field of XForms and XLSForms. CHT apps support XPath from the [ODK XForm spec](https://getodk.github.io/xforms-spec), which is based on a subset of [XPath 1.0](https://www.w3.org/TR/1999/REC-xpath-19991116/), and is evaluated by [`medic/openrosa-xpath-evaluator`](https://github.com/medic/openrosa-xpath-evaluator). The ODK XForm documentation provides useful notes about the available [operators](https://getodk.github.io/xforms-spec/#xpath-operators) and [functions](https://getodk.github.io/xforms-spec/#xpath-functions). Additionally, [CHT specific functions](#cht-xpath-functions) are available for forms in CHT apps.

{{% alert title="Note" %}} 
The `+` operator for string concatenation is deprecated and will be removed in a future version. You are strongly encouraged to use the [`concat()`](https://getodk.github.io/xforms-spec/#fn:concat) function instead. 
{{% /alert %}}
```

**Review by abbyad — CHANGES_REQUESTED** [review]
Nice additions Nick. I made suggested edits in the comment to update some older text in that section.

**Review by abbyad — APPROVED** [review]

**Review by garethbowen — APPROVED** [review]
Merge when ready.

**Pull request merged**